### PR TITLE
SG-30397: Add banner on developer.shotgridsoftware.com repository and help site

### PR DESCRIPTION
**Pull request by minna-feng**

### CODE REVIEW

**Review by NorberMV — APPROVED** [review]
This looks good ! just one suggestion, I think would be useful to generate a preview first by updating the revision value of the theme in this [ PR ](https://github.com/shotgunsoftware/tk-doc-generator/pull/52) to the latest commit here.

File: _sass/content.scss [hunk 1]
```diff
@@ -112,3 +112,57 @@
...
+  -webkit-box-pack: justify;
+  justify-content: space-between;
+  margin: calc(15px) 1rem 0 1rem;
+  overflow: hidden;
+}
+
+message {
+  display: block;
+  min-width: 200px;
+  max-width: 1200px;
```
[the remark this answers is not in the document]
Comment (carlos-villavicencio-adsk) [reply _sass/content.scss:165]: This is not going to look great on a windows size between 745px and 1150px

File: _sass/content.scss [hunk 2]
```diff
@@ -112,3 +112,57 @@
     font-weight: 500;
   }
 }
+
+banner {
```
Comment (carlos-villavicencio-adsk) [inline _sass/content.scss:116]:
```suggestion
banner {
  width: 100%
```

To be added with my previous suggestion

File: _sass/content.scss [hunk 3]
```diff
@@ -112,3 +112,56 @@
...
+  width: 100%
```
Comment (carlos-villavicencio-adsk) [inline _sass/content.scss:117]:
```suggestion
  width: 100%;
```

Sorry about that 😭 , @shaynacohen

**Review by shaynacohen — APPROVED** [review]

**Pull request merged**